index.html

```diff
@@ -13,28 +13,45 @@ <h1>Frontend Dev Quiz</h1>
     </header>
     <main>
       <section class="question-card">
-        <p class="question-text">What property flips the axes in flexbox?</p>
+        <p class="question-card__p">What property flips the axes in flexbox?</p>
         <!-- accessibility options for buttons like aria will be added with later branch -->
-        <button class="btn-answer">Show Answer</button>
-        <div class="btn-group">
-          <button class="btn-category">#html</button>
-          <button class="btn-category">#flexbox</button>
-          <button class="btn-category">CSS</button>
+        <button
+          type="button"
+          class="question-card__button"
+          aria-label="Show answer"
+        >
+          Show Answer
+        </button>
+        <div class="question-card__div-span-group">
```

Comment:
Same here, see above.

```suggestion
        <div class="question-card__categories">
```